scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -45,6 +45,7 @@ apt_arm_packages:
   - "libc6-dev:arm64"
   - "libssl-dev:arm64"
   - "libfdt1"
+  - "netcat"
```

Comment:
Is the netcat dependency only needed for ARM64 ?

Reply:
On a related note, is this a hard dependency for QEMU or something else? This is still a smaller list than documented by docs/GettingStarted/OP-TEE/Debugging/QEMU.md so I'm just trying to get an intuition for why this addition is necessary & sufficient.

Reply:
The list in the document you referenced builds the emulator from source, whereas for CI, we use a pre-built one to avoid incurring the additional compile time (about 30' on a six-core system).

`netcat` (`nc`) is used in the Jenkins task as modified by #2527* to wait for the emulator to have created the TCP endpoint in the host used to forward the guest's port 22 for its SSH server. If we don't wait, even if it's less than a second, the script fails.

* `.jenkins/Jenkinsfile`, line 104.

Reply:
@HernanGatta Got it, thanks for the explanation!

Reply:
@HernanGatta:
> Having said that, it might be a good idea to include those dependencies in the Ansible role so that by the time anyone tries to build the emulator from scratch, the dependencies will already be installed. @CodeMonkeyLeet, what do you think?

I'll defer this to @BRMcLaren but my intuition is that we should treat them as separate configurations (i.e. use pre-built vs. build their own QEMU) and we should simply support the common path; we can always add them later and I imagine the share of folks wanting to take the extra time to build QEMU (did you mean seconds or minutes?) is probably lower in general.